webpack.config.js

```diff
@@ -111,6 +111,7 @@ const nodeModulesToTranspile = [
 	'd3-scale/',
 	'debug/',
 	'@github/webauthn-json/',
```

Comment:
Out of curiosity, can you auto detect it?

Reply:
I'm not sure if it's feasible to auto-detect if a module uses ES5 or ES6+ during build time, for every module, given the sheer amount of Calypso dependencies. I believe that's why we're not simply transpiling everything, and have this list instead.